### ISSUE TITLE
Framebuffer request ID is incorrect

### ISSUE THREAD

**Issue by Qix-** [Jess4Tech/limine-protocol#3]
Perhaps it's for an older version of Limine (not sure) but the ID in the `FramebufferRequest` is incorrect:

https://github.com/Jess4Tech/limine-protocol/blob/e4982bd7a9f9a3d197d33812a0f19550d210df14/src/requests/framebuffer.rs#L9

The ID should be 

```c
#define LIMINE_FRAMEBUFFER_REQUEST { LIMINE_COMMON_MAGIC, 0x9d5827dcd881dd75, 0xa3148604f6fab11b }
```

Was getting a null response while all others work, so I'm assuming this is just a matter of outdated IDs or something. For anyone else facing this, manually specifying the ID works:

```rust
#[used]
static FB_REQUEST: Request<FramebufferRequest> = FramebufferRequest {
	id: [0xc7b1dd30df4c8b88, 0x0a82e883a194f07b, 0x9d5827dcd881dd75, 0xa3148604f6fab11b],
	revision: 0,
	..FramebufferRequest::new()
}
.into();
```

**Comment by Jess4Tech** [comment]
I am unsure of how that one happened. It might be from an older version after all, since I didn't find another ID with that value after a quick search through the header file. Thanks for the catch, fixed in 0.5.0!

**Comment by Qix-** [comment]
Most likely an old version of the framebuffer request, as per the Limine dev:

![image](https://github.com/Jess4Tech/limine-protocol/assets/885648/57bc5836-70e3-4c6d-8160-0e31db655ae3)

Thanks!
